Dockerfiles/midstream/run_tests.py

```diff
@@ -42,24 +41,61 @@ def run_validate_operator_bundle(self):
                     -e 'work_dir={work_dir}'".format(operator_dir=self.operator_dir,
                                                     operator_work_dir=self.operator_work_dir,
                                                     work_dir=self.work_dir)
-        result = subprocess.run(exec_cmd, shell=True)
+        result = subprocess.run(exec_cmd, shell=True, capture_output=True)
         return result
 
     def test_for_extract_and_validate_bundle_image(self):
+
+        self.setUp()
+        global exit_code
+        exit_code = 0
+        #check if IMAGE_TO_TEST is defined, return exit_code 85 in case it's not
+        if(self.image_to_test == None):
```

Comment:
Python expect space after if  in pep8.

Please also fix most obvious pep8 violation on other places as well.
I would ignore line too long for now.

```
sh$ git log -p -2 | pycodestyle-3 --diff | grep -v "E501 line too long"
./Dockerfiles/midstream/run_tests.py:9:1: E302 expected 2 blank lines, found 1
./Dockerfiles/midstream/run_tests.py:42:53: E128 continuation line under-indented for visual indent
./Dockerfiles/midstream/run_tests.py:43:53: E128 continuation line under-indented for visual indent
./Dockerfiles/midstream/run_tests.py:52:9: E265 block comment should start with '# '
./Dockerfiles/midstream/run_tests.py:53:31: E711 comparison to None should be 'if cond is None:'
./Dockerfiles/midstream/run_tests.py:59:60: W291 trailing whitespace
./Dockerfiles/midstream/run_tests.py:60:75: W291 trailing whitespace
./Dockerfiles/midstream/run_tests.py:93:1: E305 expected 2 blank lines after class or function definition, found 1
./Dockerfiles/midstream/run_tests.py:94:21: E231 missing whitespace after ','
./Dockerfiles/midstream/run_tests.py:94:24: E231 missing whitespace after ','
./Dockerfiles/midstream/run_tests.py:94:27: E231 missing whitespace after ','
./Dockerfiles/midstream/run_tests.py:94:30: E231 missing whitespace after ','
./Dockerfiles/midstream/run_tests.py:94:34: E231 missing whitespace after ','
./Dockerfiles/midstream/run_tests.py:96:33: E231 missing whitespace after ','
./Dockerfiles/midstream/run_tests.py:100:42: E231 missing whitespace after ','
./Dockerfiles/midstream/unit_tests.py:29:1: W293 blank line contains whitespace
./Dockerfiles/midstream/unit_tests.py:54:1: W293 blank line contains whitespace
./Dockerfiles/midstream/unit_tests.py:66:1: W293 blank line contains whitespace
./Dockerfiles/midstream/unit_tests.py:77:1: E305 expected 2 blank lines after class or function definition, found 1
```

Reply:
Run through pep8, changed code accordingly.